source/examples/functions/MceStartStop/source.iecst

```diff
@@ -222,53 +224,21 @@ END_CASE;
 // -----------------------------------------------------------------------------
 // outputs
 // -----------------------------------------------------------------------------
-io.bSystemReady :=
+io.bServoOn :=
   (io.nSmStartStop = 30) OR
   (io.nSmStartStop = 35) OR
   (io.nSmStartStop = 40) OR
   (io.nSmStartStop = 45) OR
+  (io.nSmStartStop = 50) OR
   (io.nSmStartStop = 60);
 
 // stop indicator
```

Comment:
perhaps this comment line can be deleted (check how it's done in other FB's, for constistency)